spec/actors/hyrax/actors/file_actor_spec.rb

```diff
@@ -34,7 +34,7 @@ class FileSetWithExtras < FileSet
     end
     it 'uses the relation from the actor' do
       expect(CharacterizeJob).to receive(:perform_later).with(file_set, String, io.tempfile.path)
-      expect(actor.ingest_file(io)).to be_truthy
+      actor.ingest_file(io)
```

Comment:
Right, doesn't matter much and we get into circular (not particularly informative) testing logic.  `CharacterizeJob.perform_later` is the last thing in the method.  We don't actually want that job to execute here, and we want to be sure it gets called, so we have an `expect` catch that.  At that point, we have assumed control of what the "return value" would be, so testing for it would be unnecessary.